README.md

```diff
@@ -373,16 +375,20 @@ will give a more in-depth description of the command and available options.
 #### Proxy Submissions
 
 The autograder allows course staff to submit on behalf of students using "proxy" submissions.
-The proxy submit command adds two parameters to the normal submission command:
+The suite of proxy commands use the following two new concepts:
 
- - `proxy-email` -- The student that the course staff is submitting on behalf of.
- - `proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
+ - `--proxy-email` -- The student that the course staff is submitting on behalf of.
+ - `--proxy-time` -- The "fudged" submission time that is automatically set to the earlier time between now and one minute before the due date.
    - If you want to set the time manually, time is the number of milliseconds from the [Unix Epoch](https://en.wikipedia.org/wiki/Unix_time).
+   We can use a [Unix Epoch time converter](https://www.epochconverter.com/) to help with the conversion.
 
-As the commands come from course staff, the submissions are not subject to submission restrictions.
-Here is an example usage of `autograder.cli.courses.assignments.submissions.proxy.submit` where
-course staff submits code (`my_file.py`) on behalf of `student@test.edulinq.org`.
-The example does not set a `proxy-time` so the command automatically sets one that does not mark the submissions as late.
+As the commands are issued by course staff, the submissions are not subject to submission restrictions.
+
+##### Submitting Source Files for a Student
+
+To proxy submit source files for a student, use the `autograder.cli.courses.assignments.submissions.proxy.submit` command.
+Here is an example where course staff submits code (`my_file.py`) on behalf of `student@test.edulinq.org`.
+The example does not set a `--proxy-time` so the command automatically sets one that does not mark the submissions as late.
```

Comment:
s/sets one/sets a proxy time/

Normally this would be fine, but there are a lot of new things flying around.
So, I just want to make it a little more clear.